index.html

```diff
@@ -6,8 +6,41 @@
     <title>Quiz App</title>
     <link rel="stylesheet" href="styles.css" />
   </head>
-  <header>
+  <header class="position-boxes">
     <h1>Quiz App</h1>
   </header>
-  <body></body>
+  <body>
+    <main class="position-boxes">
+      <article class="article">
+        <h2>
+          Which command do you need to create a border around an element with
+          css?
+        </h2>
+        <button type="button">Show Answer</button>
+        <ul class="position-tag-boxes">
+          <li class="tag-style">#html</li>
+          <li class="tag-style">#flexbox</li>
+          <li class="tag-style">#css</li>
+        </ul>
+      </article>
+      <article class="article">
+        <h2>How you can change the color from a border?</h2>
+        <button type="button">Show Answer</button>
+        <ul class="position-tag-boxes">
+          <li class="tag-style">#html</li>
+          <li class="tag-style">#flexbox</li>
+          <li class="tag-style">#css</li>
+        </ul>
+      </article>
+      <article class="article">
+        <h2>How you can change the color from text-elements?</h2>
```

Comment:
Small twister "How you can" -> "How can you" :)

Reply:
i fixed it, thank you :)